Assets/Scripts/Guests/Guest.cs

```diff
@@ -160,6 +327,9 @@ private void ChangeRoom(HotelRoom newRoom)
 	/// <param name="floodedOrSunk"></param>
 	private void HandleSinking(bool floodedOrSunk)

```

Comment:
what does the flooded or sunk name imply?
I find this bool a bit confusing

Reply:
Signals if the room is partially or fully submerged, which will kill the guest. Otherwise, it just moves them down one floor.